test/distributed/org/apache/cassandra/distributed/upgrade/MixedModeReplicationTestBase.java

```diff
@@ -32,7 +33,12 @@
  */
 public class MixedModeReplicationTestBase extends UpgradeTestBase
 {
-    protected void testSimpleStrategy(Versions.Major from, Versions.Major to) throws Throwable
+    protected void testSimpleStrategy(Versions.Major from) throws Throwable
```

Comment:
Should we maybe keep `from` and `to`, and instead use `CURRENT` to indicate we'd like to upgrade to whatever the version we have right now? 

We had a plan to switch to an external repo for upgrade tests, and won't need to modify test base code when we do this.

Reply:
Done in `UpgradeTestBase.upgradeFrom(from)`